app/controllers/users_controller.rb

```diff
@@ -222,14 +222,24 @@ def destroy
 
   def remove_help_one
     current_user.update_attribute("help_one",true)
+
+    respond_to do |format|
+      format.js {render nothing: true}
+    end
```

Comment:
You don't need the `respond_to` here. Just `render nothing: true` is enough. We don't call it any other way anyway.